src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int middleLine;
+    private final int height;
+
+    public IsoscelesTrapezoid(int height, int middleLine) {
+        this.middleLine = middleLine;
+        this.height = height;
+    }
```

Comment:
You can add `color` to constructor arguments and call `super(color)`.
This comment related to all figures